tutorials/easydeploy-gitlab-runner/index.mdx

```diff
@@ -0,0 +1,91 @@
+categories:
+  - containers
+dates:
+  validation: 2024-06-20
+  posted: 2024-06-20
+---
+
+GitLab Runner is a powerful tool that executes CI/CD jobs in GitLab.
+When hosted on Kubernetes, it leverages the scalability and resilience of container orchestration to efficiently manage and run your pipelines.
+By deploying GitLab Runner on a Kubernetes cluster, you can dynamically scale the number of runners based on demand, ensure high availability, and seamlessly integrate with your existing Kubernetes infrastructure.
+This setup enables you to optimize resource utilization, reduce overhead, and enhance the performance of your continuous integration and deployment processes.
+
+<Macro id="requirements" />
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+- Created a Scaleway Kubernetes [Kapsule](/containers/kubernetes/how-to/create-cluster/) or [Kosmos](/containers/kubernetes/how-to/create-kosmos-cluster/) cluster
+
+## Deploying the GitLab Runner application using Easy Deploy
+
+1. In the [Scaleway console](https://console.scaleway.com/), navigate to the **Kubernetes** section under **Containers**.
+2. Click the name of the cluster where you wish to deploy GitLab Runner. The **Cluster Information** tab will display.
+3. Click the **Easy Deploy** tab. The application dashboard displays.
+4. Click **Deploy Application**. The application deployment wizard displays.
+5. Choose **Application Library** to see the list of available applications.
+6. Select the **GitLab Runner** application.
+    <Message type="tip">
+        If you cannot find GitLab Runner on the first page, use the search bar or navigate through the library using the arrow buttons.
+    </Message>
+7. Optionally, customize the default configuration for GitLab Runner using [Helm Charts](/tutorials/kubernetes-package-management-helm/). If you do not require any customized configuration you can skip this step.
+8. Enter a name (e.g. `gitlab-runner`) and a Kubernetes namespace for your application. If no name is entered, GitLab Runner will be installed in the default namespace of the cluster.
+9. Click **Deploy Application** to deploy GitLab Runner on your Kubernetes cluster.
+10. Verify that the GitLab Runner is successfully installed and running:
+    ```bash
+    kubectl get pods -n default # replace "default" with the name of the Kubernetes namespace in which you have installed your GitLab Runner.
+    ```
+    You should see a pod with a name similar to `gitlab-runner-xxxxxx-xxxxx` in the `Running` state.
+
+## Configuring a GitLab CI/CD pipeline to use your Kubernetes Runner
```

Comment:
I was not able to go beyond this step.
Where is the gitlab repository? (not a pre-requisite)
How is it connected to my gitlab-runner in my k8s cluster?

Reply:
Added more steps